ksa_compliance/standard_doctypes/sales_invoice.py

```diff
@@ -107,4 +107,6 @@ def validate_sales_invoice(self, method) -> None:
             valid = False
 
     if not valid:
-        raise frappe.ValidationError()
+        message_log = frappe.get_message_log()
+        error_messages = "\n".join(log["message"] for log in message_log)
+        raise frappe.ValidationError(_(error_messages))
```

Comment:
No need to localize here because each message is localized individually (we do use `_` when calling msgprint)